src/pages/CurrenciesListPage.jsx

```diff
@@ -26,6 +28,16 @@ function CurrenciesList() {
   const [selectedValue, setSelectedValue] = useState("");
   return (
     <div>
+      <div className="breadcrumps-container">
+        <Breadcrumps variant="" page="currencies-list">
```

Comment:
empty string brings no additional value to the component, so it can be removed

```suggestion
        <Breadcrumps page="currencies-list">
```

Reply:
done